### ISSUE TITLE
Weird croping area

### ISSUE THREAD

**Issue by Hideman85** [nanxiaobei/antd-img-crop#82]
I don't know where this thing happen but this is not from the [library you use](https://www.npmjs.com/package/react-easy-crop) [lib demo](https://codesandbox.io/s/jolly-pond-40rw8?file=/src/index.js)

The image used in there test: https://img.huffingtonpost.com/asset/5ab4d4ac2000007d06eb2c56.jpeg?cache=sih0jwle4e&ops=1910_1000

Rendering from the original lib:
![image](https://user-images.githubusercontent.com/16165119/94020142-deab5d80-fdb2-11ea-85af-7c8dbb6bf126.png)

And from your lib:
![image](https://user-images.githubusercontent.com/16165119/94020239-fedb1c80-fdb2-11ea-9f23-a5d6c0a6e578.png)

**Comment by nanxiaobei** [comment]
It's a knowing issue on Safari and Firefox, and currently no time and good solution to solve it, PR welcome.
https://github.com/nanxiaobei/antd-img-crop/issues/69

**Comment by Hideman85** [comment]
Well I need it so I have two options either I do it myself with the original lib or I made change on this lib that is the best option.

Can you briefly explain to me the trade off? I mean the test sample of react-easy-crop works well on firefox so did you change something for having it working with chrome?

Let me know it will simplify my investigations.

**Comment by nanxiaobei** [comment]
The problem should be that the animation duration time of modal opening.
There is a PR here https://github.com/nanxiaobei/antd-img-crop/pull/70, but the solution is not ideal, so I didn't adopt it.

**Comment by Hideman85** [comment]
Solution:
```jsx
// [...]
    );
  }, [hasRotate, onClose, rotateVal]);

  //  Have a custom modal container
  const modalContainer = useMemo(() => document.createElement('div'), [])
  useLayoutEffect(() => {
    document.body.appendChild(modalContainer)
    modalContainer.addEventListener('animationend', ({ animationName }) => {
      if (animationName === 'antZoomIn') {
        //  Reset the frame
        setRotateVal(0.1)
        setRotateVal(0)
      }
    })
    return () => modalContainer.remove() //  Don't know if I need to remove first the listener since I destroy the element
  }, [])

  const renderComponent = (titleOfModal) => (
    <>
      {renderUpload()}
      {src && (
        <Modal
          getContainer={modalContainer}
          visible={true}
// [...]
```

**Comment by putrasurya** [comment]
also happened in 81.0.1 (64-bit).

any fix for this?

**Comment by putrasurya** [comment]
> The problem should be that the animation duration time of modal opening.
> There is a PR here #70, but the solution is not ideal, so I didn't adopt it.

it was Firefox rendering issue, fixed by re-setup the crop area size after media loaded tracked by react-easy-crop lib
https://github.com/nanxiaobei/antd-img-crop/pull/85

**Comment by labithiotis** [comment]
This isn't just firefox, has happen to users on safari and chrome.

**Comment by nanxiaobei** [comment]
Sorry, this issue lasted for a long time, because I didn't have enough time to invest in this project.
but it has been solved now, please update to `3.13.0`.
Thanks PR https://github.com/nanxiaobei/antd-img-crop/pull/85 by @putrasurya
